Modules/Filtering/FFT/test/itkComplexToComplexFFTImageFilterTest.cxx

```diff
@@ -88,6 +88,10 @@ transformImage(const char * inputImageFileName, const char * outputImageFileName
 
   using ComplexFilterType = itk::ComplexToComplexFFTImageFilter<ComplexImageType>;
   auto inverseComplexFilter = ComplexFilterType::New();
+
+  ITK_EXERCISE_BASIC_OBJECT_METHODS(inverseComplexFilter, ComplexToComplexFFTImageFilter, ImageToImageFilter);
```

Comment:
This line likely needs to be commented out as well until as resolution is reached for how to handle name checks on upcast classes per #3008

Reply:
:+1: `itkVnlComplexToComplexFFTImageFilterTest.cxx` does it as well. Done in 38e9b6d. Let's see if this passes now. Thanks for giving time to this.